Source/Core/DolphinQt2/MainWindow.h

```diff
@@ -83,6 +84,7 @@ class MainWindow final : public QMainWindow
   void InitCoreCallbacks();
 
   void StartGame(const QString& path);
+  void StartGame(std::unique_ptr<BootParameters> parameters);
```

Comment:
This should take an rvalue reference: `void StartGame(std::unique_ptr<BootParameters>&& parameters);`